### PR TITLE
#1475 Implement EXNCE

### PR DESCRIPTION
**Pull request by ersanyakit**
- What is the purpose of this Pull Request?
- What is the related issue for this Pull Request (if this PR fixes issue, prepend with "Fixes" or "Closes")?
- [x] I have added Specs
- [x] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [x] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly
- [x] I have implemented the `trade_page_url` method that links to the exchange page with the `base` and `target` passed in. If not available, enter the root domain of the exchange website.
- [x] I have verified at least **ONE** ticker volume matches volume shown on the trading page (use script below)

```
client = Cryptoexchange::Client.new
pairs = client.pairs 'exchange_name'
tickers = pairs.map do |p| client.ticker p end
sorted_tickers = tickers.sort_by do |t| t.volume end.reverse
```

### CODE REVIEW

File: lib/cryptoexchange/exchanges/exnce/services/market.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        def adapt_all(output)
+          timestamp = output['date']
+          output['data'].map do |ticker|
+            next unless ticker['pair']
+            adapt(ticker, timestamp)
+          end.compact
+        end
+
+        def adapt(output, timestamp)
```
Comment (ElvisLYC) [inline lib/cryptoexchange/exchanges/exnce/services/market.rb]: can support 24 hours `high` and `low` as well by adding:
```
          ticker.high      = NumericHelper.to_d(output['high'])
          ticker.low       = NumericHelper.to_d(output['low'])
```

**Comment by ElvisLYC** [comment]
@ersanyakit I could not verify the `24 hours volume` on EXNCE's official trading page.
( https://exnce.com/market/SPZ-ETH )
Can you please display the 24 hours volume on your trading page as well for us to verify the volume and approve this PR.

**Comment by ersanyakit** [comment]
> @ersanyakit I could not verify the `24 hours volume` on EXNCE's official trading page.
> ( https://exnce.com/market/SPZ-ETH )
> Can you please display the 24 hours volume on your trading page as well for us to verify the volume and approve this PR.

Now we are update it. You can see 24h Volume on market section.

info

File: lib/cryptoexchange/exchanges/exnce/services/market.rb (hunk 1 above)
Comment (ersanyakit) [reply lib/cryptoexchange/exchanges/exnce/services/market.rb]: its supports high and low let me add it quickly.

Comment (ersanyakit) [reply lib/cryptoexchange/exchanges/exnce/services/market.rb]: i added ticker high and low parameters.

**Comment by ElvisLYC** [comment]
@ersanyakit Merged. Thanks!

**Pull request merged**